src/components/Code.tsx

```diff
@@ -20,7 +20,7 @@ const darkColors = {
 
 const lightColors = {
   background: 'transparent',
-  text: '#000000',
+  text: '#FFFFFFF',
```

Comment:
Why would the default text color for "light" theme be white?

Reply:
The text color of the example code blocks looks the same: 
![image](https://github.com/user-attachments/assets/3fa6bb1a-526e-49b8-9500-77942709529b)
Checked some other pages and the text color is the same.

Reply:
Changing the color fixed the issue in contrast mode - originally the text in the code block looked like this: 
![image](https://github.com/user-attachments/assets/cc9f5450-95ed-4871-8e7e-e83940d11896)

Reply:
But... it still doesn't make sense to me?
Also why would the value be changed for the snapshots that aren't in contrast mode? Shouldn't those stay on an unchanged color since the snapshots were of light/dark that worked fine before?

Reply:
Some other potential lenses on this:
- Should these actually be `PlatformColor` values instead of these fixed colors?
- The logic below that selects colors based on theme only uses light/dark as a binary choice. How should that change to accommodate high contrast and high contrast variants?

Reply:
@chrisglein have changed the text color values to accomodate `PlatformColor`. By using a `SolidColorBrush` value, the color of the text should always accomodate any high contrast theme setting, per the documentation [here](https://microsoft.github.io/react-native-windows/docs/windowsbrush-and-theme#applying-a-system-accent-color-variant). I have tested this on the Desert and Aquatic theme modes, as well as in normal, and there are no visibility issues.